src/rgw/setup/rgw.py

```diff
@@ -29,7 +29,7 @@
 from cortx.utils.process import SimpleProcess
 from cortx.utils.log import Log
 from cortx.rgw.setup.error import SetupError
-from cortx.rgw.setup.rgw_start import RgwStart
+from cortx.rgw.setup.rgw_service import RgwService
```

Comment:
Does this import work even? given in rgw_service file, i still class name as RgwStart

Reply:
Done.